front/src/modules/trainschedule/components/ManageTrainSchedule/helpers/formatMargin.ts

```diff
@@ -1,20 +1,40 @@
 import type { Margin } from 'modules/trainschedule/components/ManageTrainSchedule/types';
 import type { PathStep } from 'reducers/osrdconf/types';
 
-const formatMargin = (pathSteps: PathStep[]): Margin => {
+const formatMargin = (pathSteps: PathStep[]): Margin | undefined => {
   const margins: Margin = {
     boundaries: [],
     values: [],
   };
```

Comment:
You can do this, I think it's more elegant :)
```suggestion
  const boundaries: string[] = [];
  const values: string[] = [];
```
Then you just have to update `boundaries` or `values`, and not `margins.boundaries` or `margins.values`

Reply:
Done